layouts/DocLayout.tsx

```diff
@@ -54,13 +55,13 @@ const DocLayout: React.FC<Props> = ({
             title: `${
                 frontMatter.title || '100ms Docs'
             } | 100ms - Video conferencing infrastructure for a video-first world`
-        }
+        },
+        canonical : `https://www.100ms.live/docs${  router.asPath === "/" ? "" : router.asPath.split('?')[0]}`
```

Comment:
the domain should come from an environment variable.

Reply:
fixed